environment/Dockerfile

```diff
@@ -19,5 +19,5 @@ RUN pip3 install -U --no-cache-dir \
 
 RUN pip3 install -U --no-cache-dir \
     aind-ophys-utils==0.0.7 \
-    aind-data-schema==0.22.0 \
-    pydantic==2.8.2 
```

Comment:
Re: removing pydantic
aind-data-schema depends on pydantic>=2.7, so it is not necessary to include as a dependency here, but all other capsules in the pipeline include it.